packages/graphql/src/schema/validation/custom-rules/directives/populatedBy.ts

```diff
@@ -16,18 +16,30 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import type { DirectiveNode } from "graphql";
+import { Kind, type DirectiveNode, type FieldDefinitionNode } from "graphql";
 import { parseValueNode } from "../../../../schema-model/parser/parse-value-node";
 import type { Neo4jGraphQLCallbacks } from "../../../../types";
 import { DocumentValidationError } from "../utils/document-validation-error";
+import type { ObjectOrInterfaceWithExtensions } from "../utils/path-parser";
+import { getInnerTypeName } from "../utils/utils";
 
 export function verifyPopulatedBy(callbacks?: Neo4jGraphQLCallbacks) {
-    return function ({ directiveNode }: { directiveNode: DirectiveNode }) {
+    return function ({
+        directiveNode,
+        traversedDef,
+    }: {
+        directiveNode: DirectiveNode;
+        traversedDef: ObjectOrInterfaceWithExtensions | FieldDefinitionNode;
+    }) {
         const callbackArg = directiveNode.arguments?.find((x) => x.name.value === "callback");
         if (!callbackArg) {
             // delegate to DirectiveArgumentOfCorrectType rule
             return;
         }
+        if (traversedDef.kind !== Kind.FIELD_DEFINITION) {
+            // delegate
```

Comment:
It could be nice to complete this comment, like the above so we know where we delegate to.